### PR TITLE
Create clients model

### PR DESCRIPTION
**Pull request by shravan1k**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Made a clients model file for database functions related to the clients table. Added a getByJobId function to retrieve clients that are attending a specified class and are email reminder recipients.

### Testing
How did you confirm your changes worked? 
- Called the getByJobId function and checked to see that I was receiving clients from the clients table with a job_id as the one specified and were all email reminder recipients.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

Screen Shot 2021-03-03 at 8 43 45 PM

is_recipient value of 1 means that they are email reminder recipients

### CODE REVIEW

File: app/models/clientsModel.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * 
+ * @param {*} jobId Id of the job to retrieve clients from
+ * @returns {Promise<void>}
+ */
+async function getByJobId(jobId) {
+    try {
+        return await db('clients')
+            .select('*')
+            .where('job_id', jobId)
+            .andWhere('is_recipient', true);
```
Comment (eliasfang) [inline app/models/clientsModel.js:15]: In what case would the is_recipient be 0?

Comment (shravan1k) [reply app/models/clientsModel.js:15]: is_recipient is still a boolean field, it is probably being represented as 0 or 1 in the database.

Comment (eliasfang) [reply app/models/clientsModel.js:15]: Sorry if my question wasn't clear but I meant to ask when would the is_recipient field ever be switched to false? As in why do we need the is_recipient field?

Comment (shravan1k) [reply app/models/clientsModel.js:15]: Oh, my bad. If the client doesn't want to receive class reminder emails then is_recipient would be set to false and therefore there we don't need to retrieve their email id for sending the email.

**Review by mingkaili — APPROVED** [review]

File: app/models/clientsModel.js (hunk 1 above)
Comment (eliasfang) [reply app/models/clientsModel.js:15]: Ohh okok make sense, thanks!

**Review by eliasfang — APPROVED** [review]
Nice!

**Pull request merged**